src/dist/replication/lib/replica_stub.h

```diff
@@ -100,6 +100,7 @@ class replica_stub : public serverlet<replica_stub>, public ref_counter
     void on_query_app_info(const query_app_info_request &req,
                            /*out*/ query_app_info_response &resp);
     void on_cold_backup(const backup_request &request, /*out*/ backup_response &response);
+    void on_cold_backup_clear(const backup_clear_request &request);
```

Comment:
I suggest that rename this function to `on_clear_cold_backup`(and same function in `backup_manager.h`), and rename request type as `clear_backup_request`.